Engines/Wine/Verbs/adobeair/script.js

```diff
@@ -1,4 +1,39 @@
+include(["engines", "wine", "engine", "object"]);
+include(["utils", "functions", "net", "resource"]);
+include(["utils", "functions", "filesystem", "files"]);
+include(["engines", "wine", "verbs", "luna"]);
+
 /**
-* Verb to install Uplay
-* @returns {Wine} Wine object
+* Verb to install adobeair
+* @returns {Wine} Wine object // Verify
```

Comment:
Don't use comment to discuss about your code. Github is a really better place for this

Reply:
ignore the 506c27d commited by mistake

Reply:
I'm talking about the //verify

Reply:
`* @returns {Wine} Wine object` not sure what it meant it was present on other verbs so i've added it too.. 

Sorry for amateur approach, was hoping to make it easier to finish..

Reply:
At the end, it's not that hard to test it, and it would be very useful to have it. Can you do it?

Reply:
It's ok. It's just that we will forget it if you keep it like this. So you can remove it. The documentation is good

Reply:
The documentation is good meaning?

Reply:
Yes, just remove the // Verify

Reply:
@qparis Please confirm that `@returns {Wine} Wine object` is true will remove then

Reply:
Like I said, this is perfectly normal. keep it like this, just remove the “// verify”
It refers to the “return this” at this end of the function which makes verbs chainable:

- wine is a “wine object”, you can call any verb like this: “wine.verb()”.
- because wine.verb() returns “this”, “wine.verb()” refers to the same object than “wine” so that you can continue to call another verb: “wine.verb().verb2()” and so on

Reply:
Noted